src/connectors/infinite-hits/connectInfiniteHits.ts

```diff
@@ -240,11 +241,14 @@ const connectInfiniteHits: InfiniteHitsConnector = (
         if (!hasShowPrevious) {
           return searchParameters;
         }
+
         const uiPage = uiState.page;
+
         if (uiPage) {
           return searchParameters.setQueryParameter('page', uiPage - 1);
         }
-        return searchParameters.setQueryParameter('page', 0);
+
+        return searchParameters.setQueryParameter('page');
```

Comment:
This part is not exactly true. It doesn't matter much at the moment because it will be updated once we do the modifications inside `getConfiguration` to "mount" a default value on the state that indicates that a widget "own" this part of the state.